src/background/services/balances/BalancePollingService.ts

```diff
@@ -28,13 +29,19 @@ export class BalancePollingService implements OnLock, OnAllExtensionClosed {
   async startPolling(
     account: Account,
     activeChainId: number,
-    roundRobinChainIds: number[]
+    roundRobinChainIds: number[],
+    tokenTypes: TokenType[]
   ) {
     // Stop any polling that may be occurring already
     this.stopPolling();
     // Start a new interval
     // this._preventSchedulingNextUpdate = false;
```

Comment:
I guess it can be removed

Reply:
Good point! Will do it in a follow-up PR.